### PR TITLE
fix live metrics

### PR DESCRIPTION
**Pull request by gabrieltorresgamez**

### CODE REVIEW

**Comment by ghost** [comment]
# RUN: 2023-04-21 19-48-16

## Hyperparameters

| Path   | Param   | HEAD~1   | workspace   |
|--------|---------|----------|-------------|

## Metrics

| Path   | Metric   | HEAD~1   | workspace   | Change   |
|--------|----------|----------|-------------|----------|

# Plots

TODO

**Comment by ghost** [comment]
# RUN: 2023-04-21 19-55-37

## Hyperparameters

| Path        | Param                               | HEAD~1   | workspace   |
|-------------|-------------------------------------|----------|-------------|
| params.yaml | dvclive.model_hparams.learning_rate | 0.1      | 0.01        |

## Metrics

| Path                 | Metric           | HEAD~1   | workspace   | Change   |
|----------------------|------------------|----------|-------------|----------|
| dvclive/metrics.json | test.epoch.acc   | 0.36129  | 0.22222     | -0.13907 |
| dvclive/metrics.json | test.epoch.f1    | 0.26923  | 0.12288     | -0.14636 |
| dvclive/metrics.json | test.step.acc    | 0.36129  | 0.22222     | -0.13907 |
| dvclive/metrics.json | test.step.f1     | 0.26923  | 0.12288     | -0.14636 |
| dvclive/metrics.json | train.epoch.acc  | 0.82433  | 0.38771     | -0.43662 |
| dvclive/metrics.json | train.epoch.f1   | 0.76904  | 0.20573     | -0.56331 |
| dvclive/metrics.json | train.epoch.loss | 0.50873  | 1.53642     | 1.02769  |
| dvclive/metrics.json | train.step.acc   | 0.96875  | 0.5625      | -0.40625 |
| dvclive/metrics.json | train.step.f1    | 0.95789  | 0.33694     | -0.62095 |
| dvclive/metrics.json | train.step.loss  | 0.15193  | 1.13785     | 0.98592  |
| dvclive/metrics.json | val.epoch.acc    | 0.8311   | 0.4742      | -0.35689 |
| dvclive/metrics.json | val.epoch.f1     | 0.7995   | 0.30134     | -0.49816 |
| dvclive/metrics.json | val.epoch.loss   | 0.62941  | 1.25206     | 0.62265  |
| dvclive/metrics.json | val.step.acc     | 0.71429  | 0.57143     | -0.14286 |
| dvclive/metrics.json | val.step.f1      | 0.5      | 0.27778     | -0.22222 |
| dvclive/metrics.json | val.step.loss    | 0.98827  | 1.21245     | 0.22418  |

# Plots

TODO

**Comment by ghost** [comment]
# RUN: 2023-04-21 20-02-06

## Hyperparameters

| Path        | Param                               | HEAD~1   | workspace   |
|-------------|-------------------------------------|----------|-------------|
| params.yaml | dvclive.model_hparams.learning_rate | 0.01     | 0.1         |

## Metrics

| Path   | Metric   | HEAD~1   | workspace   | Change   |
|--------|----------|----------|-------------|----------|

# Plots

TODO

**Comment by ghost** [comment]
# RUN: 2023-04-21 20-06-11

## Hyperparameters

| Path   | Param   | HEAD~1   | workspace   |
|--------|---------|----------|-------------|

## Metrics

| Path                 | Metric           | HEAD~1   | workspace   | Change   |
|----------------------|------------------|----------|-------------|----------|
| dvclive/metrics.json | test.epoch.acc   | 0.36129  | 0.39355     | 0.03226  |
| dvclive/metrics.json | test.epoch.f1    | 0.26923  | 0.34495     | 0.07572  |
| dvclive/metrics.json | test.step.acc    | 0.36129  | 0.39355     | 0.03226  |
| dvclive/metrics.json | test.step.f1     | 0.26923  | 0.34495     | 0.07572  |
| dvclive/metrics.json | train.epoch.acc  | 0.82433  | 0.85117     | 0.02684  |
| dvclive/metrics.json | train.epoch.f1   | 0.76904  | 0.80385     | 0.03481  |
| dvclive/metrics.json | train.epoch.loss | 0.50873  | 0.45813     | -0.0506  |
| dvclive/metrics.json | train.step.f1    | 0.95789  | 0.95789     | 0.0      |
| dvclive/metrics.json | train.step.loss  | 0.15193  | 0.14868     | -0.00326 |
| dvclive/metrics.json | val.epoch.acc    | 0.8311   | 0.8636      | 0.03251  |
| dvclive/metrics.json | val.epoch.f1     | 0.7995   | 0.83737     | 0.03788  |
| dvclive/metrics.json | val.epoch.loss   | 0.62941  | 0.41393     | -0.21548 |
| dvclive/metrics.json | val.step.acc     | 0.71429  | 0.85714     | 0.14286  |
| dvclive/metrics.json | val.step.f1      | 0.5      | 0.66667     | 0.16667  |
| dvclive/metrics.json | val.step.loss    | 0.98827  | 0.23321     | -0.75506 |

# Plots

TODO

**Comment by ghost** [comment]
# RUN: 2023-04-21 20-09-11

## Hyperparameters

| Path        | Param                               | HEAD~1   | workspace   |
|-------------|-------------------------------------|----------|-------------|
| params.yaml | dvclive.model_hparams.learning_rate | 0.1      | 0.01        |

## Metrics

| Path                 | Metric           | HEAD~1   | workspace   | Change   |
|----------------------|------------------|----------|-------------|----------|
| dvclive/metrics.json | epoch            | -        | 2           | -        |
| dvclive/metrics.json | step             | -        | 354         | -        |
| dvclive/metrics.json | test.epoch.acc   | -        | 0.4086      | -        |
| dvclive/metrics.json | test.epoch.f1    | -        | 0.22118     | -        |
| dvclive/metrics.json | test.step.acc    | -        | 0.4086      | -        |
| dvclive/metrics.json | test.step.f1     | -        | 0.22118     | -        |
| dvclive/metrics.json | train.epoch.acc  | -        | 0.30632     | -        |
| dvclive/metrics.json | train.epoch.f1   | -        | 0.14373     | -        |
| dvclive/metrics.json | train.epoch.loss | -        | 1.66662     | -        |
| dvclive/metrics.json | train.step.acc   | -        | 0.53125     | -        |
| dvclive/metrics.json | train.step.f1    | -        | 0.36004     | -        |
| dvclive/metrics.json | train.step.loss  | -        | 1.35932     | -        |
| dvclive/metrics.json | val.epoch.acc    | -        | 0.41979     | -        |
| dvclive/metrics.json | val.epoch.f1     | -        | 0.30185     | -        |
| dvclive/metrics.json | val.epoch.loss   | -        | 1.46609     | -        |
| dvclive/metrics.json | val.step.acc     | -        | 0.57143     | -        |
| dvclive/metrics.json | val.step.f1      | -        | 0.38333     | -        |
| dvclive/metrics.json | val.step.loss    | -        | 1.48332     | -        |

# Plots

TODO

**Comment by ghost** [comment]
# RUN: 2023-04-21 20-15-36

## Hyperparameters

| Path        | Param                               | HEAD~1   | workspace   |
|-------------|-------------------------------------|----------|-------------|
| params.yaml | dvclive.model_hparams.learning_rate | 0.01     | 0.1         |

## Metrics

| Path                 | Metric           | HEAD~1   | workspace   | Change   |
|----------------------|------------------|----------|-------------|----------|
| dvclive/metrics.json | epoch            | -        | 2           | -        |
| dvclive/metrics.json | step             | -        | 354         | -        |
| dvclive/metrics.json | test.epoch.acc   | -        | 0.39283     | -        |
| dvclive/metrics.json | test.epoch.f1    | -        | 0.28418     | -        |
| dvclive/metrics.json | test.step.acc    | -        | 0.39283     | -        |
| dvclive/metrics.json | test.step.f1     | -        | 0.28418     | -        |
| dvclive/metrics.json | train.epoch.acc  | -        | 0.83439     | -        |
| dvclive/metrics.json | train.epoch.f1   | -        | 0.78789     | -        |
| dvclive/metrics.json | train.epoch.loss | -        | 0.49025     | -        |
| dvclive/metrics.json | train.step.acc   | -        | 0.90625     | -        |
| dvclive/metrics.json | train.step.f1    | -        | 0.8537      | -        |
| dvclive/metrics.json | train.step.loss  | -        | 0.20279     | -        |
| dvclive/metrics.json | val.epoch.acc    | -        | 0.85866     | -        |
| dvclive/metrics.json | val.epoch.f1     | -        | 0.83285     | -        |
| dvclive/metrics.json | val.epoch.loss   | -        | 0.43854     | -        |
| dvclive/metrics.json | val.step.acc     | -        | 0.85714     | -        |
| dvclive/metrics.json | val.step.f1      | -        | 0.66667     | -        |
| dvclive/metrics.json | val.step.loss    | -        | 0.49829     | -        |

# Plots

TODO

**Comment by gabrieltorresgamez** [comment]
the problem doesnt lay here... the problem is somehow caused with external storage i believe
